lib/cassanity/config.rb

```diff
@@ -0,0 +1,22 @@
+
+require 'ostruct'
+require 'yaml'
+require 'singleton'
+require 'erb'
+
+module Cassanity
+  class Config < OpenStruct
+    include Singleton
+
+    CONFIG_FILE = 'config/cassanity.erb.yml'
```

Comment:
Why use a hard coded config file instead of just putting the options here in ruby? I'd rather just see them in ruby.